### ISSUE TITLE
DeviceHub exception with ASCOM.SoftwareBisque.Telescope driver

### ISSUE THREAD

**Issue by rfurmanimd** [ASCOMInitiative/ASCOMDeviceHub#3]
The "ASCOM Telescope Driver for TheSky." does not implement "DestinationSideOfPier", so doing a direct slew with the ASCOMDeviceHub UI generates a "MethodNotImplemetedException" originating from TelescopeManagerAccess.cs, DestinationSideOfPier. You can avoid this exception by daisy-chaining ASCOMDeviceHub to POTH and then connecting to TheSky telescope driver as POTH implements a heuristic, in ScopeHW.bas,  ScopeSOP, to infer SideOfPier for drivers not implementing this function.

It would be helpful to have ASCOMDeviceHub implement the POTH workaround as the TheSky telescope driver has not been updated for about 5 years.

**Comment by astroman133** [comment]
Although it is unknown why your driver does not provide this simple calculation, I addressed your issue in version 6.4.1.4. Please try this version and provide feedback. Thanks!

**Comment by rfurmanimd** [comment]
I tried the new version 6.4.1.4 with the “ASCOM driver for TheSky.” and a telescope simulator in TheSkyX without encountering any issues using the “Direct Slew” tab controls. As soon as the skies clear, I’ll try it with real equipment.

I also downloaded the source code zip package as the GitHub repository has not (yet?) updated with any file changes. I could not find any source changes in the zip sources, however, and compiling the sources with VS 2019 still exhibits the old behavior of an exception thrown by DestinationSideOfPier. Will the updated changes be available soon?

Thanks for the quick update,
Roy Furman

FYI: My driver is the “Ascom 2X Mount Adaptor (Paramount, etc., 6.1) for TheSky X and TheSky 5/6 (external link)” from the ASCOM Standards web site. This Software Bisque driver is required for the Paramount line of mounts. Unfortunately, it has not been updated in years.
